wagtail_shortcode/static_src/main.css

```diff
@@ -0,0 +1,5 @@
+/*
```

Comment:
**Note:**
Not needed but seemed like a waste to remove code that might be needed in the future?